Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
```

Comment:
Have you tried to integrate this as a Swift Package in another Sample App ? Does it complain that there is not such folder "Sources" ?

Reply:
Yes I'm using this project via Swift Package.
And it didn't complain about "Sources" folder.